backend/LfNext/LcmDebugger/Program.cs

```diff
@@ -10,8 +10,10 @@
 var app = builder.Build();
 
 var fieldWorksProjectList = app.Services.GetRequiredService<FieldWorksProjectList>();
-var fwDataProject = fieldWorksProjectList.GetProject("fruit") ?? throw new InvalidOperationException("Could not find project");
+var fwDataProject = fieldWorksProjectList.GetProject("sbe-flex") ?? throw new InvalidOperationException("Could not find project");
```

Comment:
This should probably be reverted
```suggestion
var fwDataProject = fieldWorksProjectList.GetProject("fruit") ?? throw new InvalidOperationException("Could not find project");
```